charts/prometheus-adapter/templates/custom-metrics-apiserver-psp.yaml

```diff
@@ -0,0 +1,71 @@
+{{- if .Values.psp.create -}}
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "k8s-prometheus-adapter.fullname" . }}
+  labels:
+    app: {{ template "k8s-prometheus-adapter.name" . }}
+    chart: {{ template "k8s-prometheus-adapter.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  {{- if .Values.hostNetwork.enabled }}
+  hostNetwork: true
+  {{- end }}
+  fsGroup:
+    rule: RunAsAny
+  runAsGroup:
+    rule: MustRunAs
+    ranges:
+    - max: 0
+      min: 0
+  runAsUser:
+    rule: MustRunAs
+    ranges:
+    - max: 10001
+      min: 10001
```

Comment:
Would be better to specify as min 1024 and max 65535. There's no explicit reason it has to be 10001 except that we arbitrarily hardcoded the chart to that. We'll probably want to make the security policies configurable on the chart at some point.

Reply:
Sounds good